### PR TITLE
Add FPMMTrade from trade events

### PR DESCRIPTION
**Pull request by girianshido**
Add a **FPMMTrade** to record information from `FixedProductMarketMaker` contracts on `FPMMSell` and `FPMMBuy` events.

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -169,6 +169,28 @@ type FpmmParticipation @entity {
...
+  fpmm: FixedProductMarketMaker!
+
+  # lifted from fpmm
+  title: String
+  collateralToken: Bytes!
+
+  type: TradeType!
+  creator: String!
+  creationTimestamp: BigInt!
+  amount: BigInt!
```
Comment (cag) [inline schema.graphql:188]: Maybe `collateralAmount` would be a better name?

File: schema.graphql [hunk 2]
```diff
@@ -169,6 +169,28 @@ type FpmmParticipation @entity {
...
+  feeAmount: BigInt!
+  outcomeIndex: BigInt!
+  outcomeTokensTrade: BigInt!
```
Comment (cag) [inline schema.graphql:191]: `outcomeTokensTraded` corresponds better with `outcomeTokensBought` and `outcomeTokensSold`.

**Review by cag — APPROVED** [review]
You can take the renaming suggestions if you want, but logically, everything looks great. Thanks!

File: schema.graphql (hunk 1 above)
Comment (davidalbela) [reply schema.graphql:188]: Will add your suggestions! Thanks @cag !

File: schema.graphql (hunk 2 above)
Comment (cyberosa) [reply schema.graphql]: In the README of this repo, the only name that appears is `outcomeTokensBought`, however when I try to query a trade requesting that field I get an error 
`"message": "Cannot query field \"outcomeTokensBought\" on type \"FpmmTrade\". Did you mean \"outcomeTokensTraded\"?",`

Someone renamed the README but the schema is the same as the old one. Could someone then explain to me what is supposed to be outcomeTokensTrade? Is the amount of tokens bought?
Thanks

**Pull request merged**